### ISSUE TITLE
Tag released versions on Git

### ISSUE THREAD

**Issue by omasanori** [mwmbl/crawler-extension#39]
It improves the transparency and traceability, which is especially important for client-side components.

**Comment by ColinEspinas** [comment]
For sure something that would improve the current release flow.

I am not sure about how @daoudclarke does releases but if I'm not wrong, it is not linked at all to any git tag or version.

I mentioned in a, not so recent, issue (#5) the improvement that using automations could bring benefits to the current flow. A first step in the right direction would indeed be to create release tags for each version.

**Comment by daoudclarke** [comment]
Yes, this is a great idea, and we should totally automate it using GitHub actions.
